src/index.js

```diff
@@ -115,7 +138,11 @@ const getLikedPosts = (timestamp) => {
           for (let i = 0; i < likedPosts.length; i++) {
             const currentPost = likedPosts[i];
             if (Array.isArray(currentPost.photos)) {
-              currentPost.photos.forEach(pushToArray);
+              const filebase = cleanFileName(`${currentPost.blog_name}_${currentPost.timestamp}`);
```

Comment:
Agree. Let's add that as a new property for the config object called `attribution`.

People would need to:
```javascript
tumblrLksDownldr.setGlobalParams(
  {
    url: 'yourblog.tumblr.com',
    ...
    attribution: true,
    ...
  }
);
tumblrLksDownldr.getLikedPosts();
```

What do you think?